libmachine/drivers/base_test.go

```diff
@@ -0,0 +1,31 @@
+package drivers
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestIP(t *testing.T) {
+	ip, err := (&BaseDriver{}).GetIP()
```

Comment:
Let's use a test table here, it'd be clearer and will show newcomers the right way how to do such a test, i.e.:

```go
	cases := []struct {
		baseDriver  Driver
		expectedIp  string
		expectedErr error
	}{
		{&BaseDriver{}, "", errors.New("IP address is not set")},
		{&BaseDriver{IPAddress: "2001:4860:0:2001::68"}, "2001:4860:0:2001::68", nil},
		{&BaseDriver{IPAddress: "192.168.0.1"}, "192.168.0.1", nil},
		{&BaseDriver{IPAddress: "192.168.0.1"}, "192.168.0.1", nil},
		{&BaseDriver{IPAddress: "::1"}, "::1", nil},
		{&BaseDriver{IPAddress: "whatever"}, "", errors.New("IP address is invalid: whatever")},
	}

	for _, c := range cases {
		ip, err := c.baseDriver.GetIP()
		assert.Equal(t, ip, c.expectedIp)
		assert.Equal(t, err, c.expectedErr)
	}
```